views/forms/LoginForm/LoginForm.test.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import LoginForm from '.'
+import FormProviderFixture from '../../../test/FormProviderFixture'
+
+describe('Login Form', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = render(<LoginForm />, { wrapper: FormProviderFixture })
+  })
+
+  test('renders the form', () => {
+    const { container } = wrapper
+    const form = container.firstChild
+
+    expect(form).toBeInTheDocument()
+  })
```

Comment:
same comment re: using screen and this test may not be valuable given the others.

Reply:
[1d6bb22](https://github.com/AMP-SCZ/dpdash/pull/468/commits/1d6bb2230345cbb84bf5aac92821f916e7aaad79)